### PR TITLE
grant/email: addded change owner template

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

Demo: https://gitcoincore.slack.com/files/UAW6336F2/FEYNYNGR3/_derp.mov

```
- `A` -> changes grant ownership to `B` 
- we keep record of that and send a mail to `B` 
- `B` can choose to accept or reject
- If `B` accepts -> he is redirected to grant page with alert say he has rejected the request & a mail gets sent to him saying the same
- If `B` rejects -> he is redirected to grant page with alert notifying he is now the owner & a mail gets sent to `A` saying the change was rejected
```

For @mbeacom / @captnseagraves 

I added a new field in model `request_ownership_change` which gets populates when owners tries to change owner. Once the potentail new owner accepts / rejects  -> we update the `admin_profile` and set `request_ownership_change` as None

- [ ] verify if the email calls are done right 
    [request new owner](https://github.com/gitcoinco/web/pull/3252/files#diff-9c83a1892e5b433bc22483a941276b94R145) 
   [new owner rejects](https://github.com/gitcoinco/web/pull/3252/files#diff-9c83a1892e5b433bc22483a941276b94R184)
   [new owner accepts](https://github.com/gitcoinco/web/pull/3252/files#diff-9c83a1892e5b433bc22483a941276b94R179)
- [ ] pass this [param](https://github.com/gitcoinco/web/pull/3252/files#diff-9c83a1892e5b433bc22483a941276b94R146) to the redirect so that [this alert](https://github.com/gitcoinco/web/pull/3252/files#diff-405c08ff03effe01b34326d2175ff1beR374) is rendered notifying the current owner that an email has been sent out 
- [ ] should this [change ownership approval mail](https://github.com/gitcoinco/web/pull/3252/files#diff-9c83a1892e5b433bc22483a941276b94R178) be sent to the previous owner ?

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -181,6 +189,13 @@ def new_grant(request):
     return HttpResponse(response_html)
 
 
+@staff_member_required
+def change_grant_owner(request):
+    grant = Grant.objects.all().first()
```
Comment (mbeacom) [inline app/retail/emails.py]: `grant = Grant.objects.first()` - You can drop the `.all()` here

**Comment by thelostone-mc** [comment]
@mbeacom updated that and the other places where it was declared as that. thanks 🙌

**Comment by mbeacom** [comment]
Looks like this wasn't merged before `grantz-cleanup` was merged to `master` and `stable` - 😭

File: app/retail/emails.py [hunk 2]
```diff
@@ -136,51 +160,72 @@ def successful_contribution(request):
...
+    grant = Grant.objects.first()
     subscription = Subscription.objects.filter(grant__pk=grant.pk).first()
     response_html, __, __ = render_support_cancellation_email(grant, subscription)
     return HttpResponse(response_html)
 
 
 @staff_member_required
 def thank_you_for_supporting(request):
-    grant = Grant.objects.all().first()
+    grant = Grant.objects.first()
```
Comment (mbeacom) [inline app/retail/emails.py]: My hero

**Comment by thelostone-mc** [comment]
@mbeacom / @captnseagraves mesa still can't figure this out 🤷‍♂️

**Comment by thelostone-mc** [comment]
Dup of https://github.com/gitcoinco/web/pull/3366

**Pull request closed without merging**